api/api/auth/tests.py

```diff
@@ -159,6 +160,7 @@ def test_createToken_invalidAuthorizationCodeInRequest_throwsForbiddenError(
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    @pytest.mark.django_db
```

Comment:
This test doesn't use a database.

Reply:
here it does
`tokens_count = Token.objects.count()`
`users_count = User.objects.count()`